### DISCUSSION TITLE
Mi pregunta

### DISCUSSION THREAD

**Discussion by AbiKorzyk** [Launch-X-Latam/MisionFrontEnd#12] [Q&A]

**Comment by JP0364** [comment]
https://youtu.be/OIE9r0J1iRk
https://youtu.be/8B_qtbdlLSU
El equipo de Innovacción hizo algunos tutoriales para entender la plataforma

**Reply by AbiKorzyk** [reply]
Gracias :D

**Reply by AldrickSalazar** [reply]
si, pero me referia a Javascript, css o java

**Comment by JosafatJimenezB** [comment]
Solo aprieta el boton de fork para crear un copia para ti y a esta le puedes editar todo lo que quieras, esta siempre se va a actualizar al contenido que suba el mission commander

**Reply by AbiKorzyk** [reply]
Gracias ;D

**Reply by JosafatJimenezB** [reply]
denada compañera explorer! :sparkles:

**Reply by datavasis** [reply]
Hola, no me queda muy claro como funciona lo de los forks, ¿Si se actualiza mi copia conforme se actualice el repositorio original? Pensé que solo era una clon, pero no actualizable.

**Reply by Cerecero** [reply]
@dacobot Si es una copia del repo original, pero esta se actualiza cuando tu lo deseas no se hace de manera automatica (a menos que tu lo configures de esa manera).

**Comment by ProDeveloperRoy** [comment]
El fork te sirve practicamente para copiar el repositorio ajeno al tuyo y poder hacer con el lo que quieras :)

**Reply by AbiKorzyk** [reply]
Eso si lo sabía, solo que no tenía muy claro como hacerlo para tenerlo guardado
